# Go To Sleep Command

bool sleep
---
